repo.go

```diff
@@ -625,10 +625,12 @@ func (r *Repository) downloadAndSaveItem(client Client, downloadingItem *downloa
 	downloadingItem.path = r.fullPath(it.filePath)
 	downloadingItem.pathMu.Unlock()
 
-	// try a few times in case of network trouble
 	var h hash.Hash
 	var x *exif.Exif
-	for i := 0; i < 3; i++ {
+
+	attempt := 1
+
+	for {
```

Comment:
As for me, it's better to get into an infinite loop than download a garbage in place of valid data in backup software. :) Ok, I rewrote the loop as you requested.